### DISCUSSION TITLE
Easy way to connect the same line to another node

### DISCUSSION THREAD

**Discussion by HubKing** [Comfy-Org/ComfyUI_frontend#1031] [Ideas]
For example, see the example below. the "model" connection to the "KSampler" is from a "Load Checkpoint" node on the far left. The node is out of the view, so to view that node, I have to scroll to the left. 

Now, if I want to add a new "KSampler" node and connect the same model to it, it seems that I have to scroll the view to the left, and click the "MODEL" of the "Load Checkpoint". This seems inconvenient.

Could there be an easy way to duplicate the connection? For example, I could shift drag the "model" node of the old "KSampler" to the "model" node of the new "KSampler", and instead of moving the existing connection, shift dragging could duplicate the connection (i.e., both "KSampler"s are connected to the same "Load Checkpoint").

![image](https://github.com/user-attachments/assets/b5972010-4b42-4465-9090-c982ca5870ff)

I had asked it here https://github.com/comfyanonymous/ComfyUI/discussions/5079 , but he told me to ask it here.

**Comment by LukeG89** [comment]
They recently added the new shortcut `shift + drag` to copy existing connections (https://github.com/Comfy-Org/ComfyUI_frontend/pull/841), but unfortunately it doesn't work on Chrome for some reason.

**Reply by HubKing** [reply]
Hah. Funny that the developer thought of the exact same shortcut, because shift + drag was just an example that I came up with after starting to write this post. I am using FireFox, so I check the version and my version was a little bit lower than that. So, all I have to do is wait until the updated version gets released.

**Reply by LukeG89** [reply]
If you don't want to wait, you can already use the latest frontend version by using this argument:
```
--front-end-version Comfy-Org/ComfyUI_frontend@latest
```
I would like to warn you that yesterday they updated to version 1.3.x and many things have changed.
